js/manager.js

```diff
@@ -150,11 +148,18 @@ export const DFPManager = Object.assign(new EventEmitter(), {
 
   registerSlot({ dfpNetworkId, adUnit, sizes, renderOutOfThePage, sizeMapping,
                  targetingArguments, slotId, slotShouldRefresh }) {
-    if (!registeredSlots.hasOwnProperty(slotId)) {
-      registeredSlots[slotId] = { slotId, sizes, renderOutOfThePage,
-                                  dfpNetworkId, adUnit, targetingArguments,
-                                  sizeMapping, slotShouldRefresh, loading: false,
-                                };
+    if (!registeredSlots[slotId]) {
```

Comment:
This also conflits with your ESLint config that requires us not to use that on an object. Since slotId is always a string or undefined, I think an existence check suffices. But we can relax if you want.